enhancements/ingress/set-delete-http-headers.md

```diff
@@ -782,6 +765,26 @@ For this reason, the `host` header is in the deny list of headers that are prohi
 
 Please note that a custom `host` header may be specified via RouteSpec. Please refer to `Open Questions` for more details.
 
+### Cookie and Set-Cookie
+There are three reasons we blocked `Cookie` and `Set-Cookie`:
+
+1. There were no use cases mentioned for these headers in [RFE](https://issues.redhat.com/browse/RFE-464).
+
+2. The cookies that HAProxy sets are used for session tracking, to map a client connection to a particular 
+    backend server.
+
+   The cookie name is an MD5 hash value derived from the backend's name (similarly, the cookie value is a hashed value
+   derived from the backend server's endpoint). 
+   
+   So the cookie names that we would want to forbid are all the possible values
+   of the MD5 hashing algorithm, but it's extremely unlikely that someone would accidentally choose the same cookie name
+   that HAProxy used. 
```

Comment:
These lines don't add any meaning to the discussion.

Reply:
but we need to explain what the interference may be right ?

Reply:
Your comment makes it look like the whole thing is "extremely unlikely" to happen, so I don't think this is the interference we're concerned with.

Reply:
yes removed